lms/djangoapps/instructor/views/instructor_dashboard.py

```diff
@@ -47,13 +48,14 @@
 @cache_control(no_cache=True, no_store=True, must_revalidate=True)
 def instructor_dashboard_2(request, course_id):
     """ Display the instructor dashboard for a course. """
-    course_key = SlashSeparatedCourseKey.from_deprecated_string(course_id)
+    course_key = CourseKey.from_string(course_id)
     course = get_course_by_id(course_key, depth=None)
```

Comment:
I know this isn't part of your PR, but I'm surprised this code doesn't handle the case in which (a) the course ID is invalid, or (b) no course exists for the given ID.